stock_prices/stock_prices.py

```diff
@@ -3,13 +3,58 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
+    index = len(prices)
+    highest = -999999999999
+    while index >0:
```

Comment:
this is looking good, but you're calling the for loop `n` times, once for every time your index runs down until it breaks out of the while loop. 
when you're done with MVP, see if you can get this to O(n) instead!